server2/Arcadia.Assistant/Arcadia.Assistant.Employees.Contracts/EmployeeMetadata.cs

```diff
@@ -59,6 +59,8 @@ public EmployeeMetadata(EmployeeId employeeId, string email)
             return CalculateYearsFromDate(this.BirthDate, date);
         }
 
+        public string Name => $"{this.FirstName ?? string.Empty} {this.LastName ?? string.Empty}";
```

Comment:
In case if first name or last name are empty property will contain leading or trailing spaces.

Reply:
Fixed.